public/pages/MonitorDetails/containers/MonitorDetails.js

```diff
@@ -484,18 +484,9 @@ export default class MonitorDetails extends Component {
         {this.renderNoTriggersCallOut()}
         <EuiFlexGroup alignItems="flexEnd">
           <EuiFlexItem grow={false}>
-            <EuiTitle size="l" style={{ whiteSpace: 'nowrap', overflow: 'hidden' }}>
-              <h1
-                style={{
-                  whiteSpace: 'nowrap',
-                  maxWidth: '90%',
-                  textOverflow: 'ellipsis',
```

Comment:
Should we keep this ellipsis style? Mainly problematic if this doesn't wrap and monitor name is long

Reply:
Actually the ellipsis style wasn't even applied here today. Here's the generated html based on existing code(without my change), and you can see in `<h1>`, the inline style isn't there. I assume that there's some overwrite of parent element style.
![image](https://github.com/user-attachments/assets/1362bd05-c362-4941-8692-624e7f2facd2)

Reply:
weird, not sure why it has nowrap :-/ but you're not making anything worse, so its fine